### ISSUE TITLE
Request - TACC Login Help NHSM - Volunteer Verification

### ISSUE THREAD

**Issue by Kmullineaux** [ArctosDB/arctos#9187]
**Describe what you're trying to do** 

Hello all, I've submitted a ticket to TACC regarding the issue of Gmail/volunteer accounts without institutional emails, but I wanted to see if the Arctos team could help on the back end. 

I've been away this past week and am only seeing the issue this morning. Sorry for the late entry into this conversation. I saw another issue posted with the same/similar problem (TACC login NMMNHS #9173), and perhaps more progress has been made there. I didn't want to confuse that conversation since it was institution-specific, but if I can get any help as well, that would be great.

I have interns and volunteers who are not assigned institutional emails as a matter of policy but who use Arctos as a main function of their work. I have new interns every semester and rotating volunteers, so I'm sure this will come up again. 

Current usernames include: 

pholobaugh
afriedman1
hglavan
rbaldwin1

I'm also currently having trouble getting into my account using TACC MFA, as my account has been labeled pending, but I submitted a separate ticket to TACC about that. Thanks for your help!

**Comment by Kmullineaux** [comment]
@mkoo I am following up on this one to see if there have been any changes or progress on the TACC verification issue. It seems that one of my volunteers (pholobaugh) has been able to keep working in Arctos despite not having the correct email through Tacc (not sure how), but my other volunteer, sgonzaga (who has been unable to work lately but is back now), has not been able to get through the verification process. 

Thanks!

**Comment by wellerjes** [comment]
I'm having this same issue; I don't know why it's not consistent among users.

**Comment by Kmullineaux** [comment]
@wellerjes Just yesterday, the person who still had access got kicked out - so I think it might just be delayed for some people, unfortunately.

**Comment by mkoo** [comment]
I forgot to say here that I did send your two volunteers (pholobaugh, sgonzaga ) recently but never know if they do anything with that info!

@wellerjes what is the user name with issues?

**Comment by Kmullineaux** [comment]
@mkoo I submitted a ticket with Tacc as you suggested, and I got the following response: Hello Kat,

Unfortunately, we are no longer able to support gmail accounts. Because we are
largely supported by NSF funds we have to abide by the restrictions set out by
the federal government which includes the ability to identify a user in
association with an institution. If you are not able to provide them with a
Maryland Nature email address then I recommend reaching out to Arctos directly
to see if they are willing to support providing an email address.
Alternatively, several universities have the ability to request an EID and an
affiliated email address which they could use. Example from The University of
Texas: https://idmanager.its.utexas.edu/eid_self_help/?createEID&
qwicap-page-id=9E06C05A6C3729F6

We're sorry for the inconvenience.
--
Virginia Trueheart, MSIS
Director, User Services
Subscribe to user news: https://portal.tacc.utexas.edu/news/subscribe

**Comment by wellerjes** [comment]
> [@wellerjes](https://github.com/wellerjes) what is the user name with issues?

The username with issues is mdraftz; the username without issues (yet) is plinge

Thanks!

**Comment by dustymc** [comment]
Done?

**Comment by Kmullineaux** [comment]
@dustymc Yes, done!
